src/components/App.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import '../styles/App.css';
+import Header from './Header'
+import { Switch, Route } from 'react-router-dom';
+import HospitalTable from "./HospitalTable";
+import CreateHospital from "./CreateHospital";
+import EditHospital from "./EditHospital";
+
+function App() {
+  return (
+      <div className="center w85">
+        <Header />
+        <div className="ph3 pv1 background-gray">
+          <Switch>
+            <Route exact path="/viewHospitals" component={HospitalTable} />
```

Comment:
Once we start building out the app, maybe we should nest these all under one route - for example, `/hospitals/view`, `/hospitals/edit`, `/hospitals/create`, etc

Reply:
that's a good idea for sure, but this will also depend on the structure of our app in general - i'm leaning towards full screen modals instead of specific routes as a better solution